### DISCUSSION TITLE
Change only samba password

### DISCUSSION THREAD

**Discussion by rmveras** [ltb-project/self-service-password#497] [Q&A]
Hi guys,

Is there a way to change only the sambaNTPassword attribute?

**Comment by coudot** [comment]
No. What is the use case?

**Reply by rmveras** [reply]
The userPassword attribute is synchronized from a central LDAP server and we don't want to change it, locally, we use sambaNTPassword to access the file server.

**Reply by coudot** [reply]
So in SSP, you can't just change samba password. You could do it by hacking the change_password method code.

Why is sambaNTPassword not changed on your central LDAP?

**Reply by rmveras** [reply]
Because there is no such attribute there. The central server is not managed by me.
I will try to change the method. 
Thanks @coudot
